examples/chart/teleport-kube-agent/values.schema.json

```diff
@@ -279,6 +279,11 @@
                     "$id": "#/properties/tls/properties/existingCASecretName",
                     "type": "string",
                     "default": ""
+                },
+                "existingCASecretKeyName": {
+                    "$id": "#/properties/tls/properties/existingCASecretKeyName",
+                    "type": "string",
+                    "default": ""
```

Comment:
Should the default also be set to `ca.pem` like in the other charts, and also add `existingCASecretKeyName` to `required`?